iam-login-service/src/main/java/it/infn/mw/iam/api/scim/provisioning/ScimGroupProvisioning.java

```diff
@@ -168,6 +188,11 @@ public ScimGroup replace(String id, ScimGroup scimItemToBeReplaced) {
     updatedGroup.touch();
 
     groupRepository.save(updatedGroup);
+
+    eventPublisher
+      .publishEvent(new GroupReplaceEvent(this, updatedGroup, existingGroup, String.format(
+          "Replaced group %s with new group %s", existingGroup.getName(), updatedGroup.getName())));
```

Comment:
I don't like very much this message, but I haven't a good proposal.
It's not much that the group is being replaced, is more some information is being updated.